diff --git a/node_modules/@expo/cli/build/src/start/server/metro/TerminalReporter.js b/node_modules/@expo/cli/build/src/start/server/metro/TerminalReporter.js
index 38c75be..9bafea9 100644
--- a/node_modules/@expo/cli/build/src/start/server/metro/TerminalReporter.js
+++ b/node_modules/@expo/cli/build/src/start/server/metro/TerminalReporter.js
@@ -29,7 +29,7 @@ function _chalk() {
     return data;
 }
 function _TerminalReporter() {
-    const data = /*#__PURE__*/ _interop_require_default(require("metro/src/lib/TerminalReporter"));
+    const data = /*#__PURE__*/ _interop_require_default(require("metro/private/lib/TerminalReporter"));
     _TerminalReporter = function() {
         return data;
     };
